pkg/controllers/node/termination/suite_test.go

```diff
@@ -76,6 +78,9 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("Termination", func() {
+	// expectedReconciliations is the number of reconciliations the termination controller should need to remove a node.
+	// One reconciliation to start terminating the nodeclaim, and another to get the instance, see that it's gone, and remove the node.
+	const expectedReconciliations = 2
```

Comment:
IMO: I think abstracting in this way may actually make things more confusing than just calling the ExpectObjectReconciled function multiple times in code. This isn't that strange of a pattern to do (we do it in the NodeClaim termination loop and in some other termination flows) and, in my opinion, creates a lot less indirection when it comes to figuring out what's happening and _why_ it's happening in a particular test case.

What do you think about just calling ExpectObjectReconciled twice and then just commenting above each call what the call is there for?

Reply:
Sure, I don't have a strong preference either way. My original thought on eventually was so we could decouple the number of iterations, but if we're doing a set number either way it doesn't make a big difference. I think I agree that it's more readable if we just inline things.